src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -681,6 +686,11 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 		return neighbourGroup;
 	}
 
+	private newSideRightGroup(): IEditorGroup {
```

Comment:
```suggestion
	private newSideGroup(): IEditorGroup {
```

Reply:
See my comment below explaining a new approach. But could change it to simply newEditorGroup.